supported_build_configurations.yaml

```diff
@@ -35,7 +35,7 @@ test-blinky:
   - science
 
 test-blockingneo:
-#  - nucleo
+  #  - nucleo
```

Comment:
but wai

Reply:
idk why but when I revert it (by removing the space) and save it defaults to the indented version. Is there a way to fix this?